remix.init script failed

I tried to use the supa-fly-stack template but it failed with the message shown below.  
I'm running on Windows 10
npx --version = 10.2.5
node --version = v20.10.0

```shell
> npx create-remix --template rphlmr/supa-fly-stack
 remix   v2.4.0 💿 Let's build a better website...

   dir   Where should we create your new project?
         ./appname

      ◼  Template: Using rphlmr/supa-fly-stack...
      ✔  Template copied

   git   Initialize a new git repository?
         Yes

  deps   Install dependencies with npm?
         Yes

  init   This template has a remix.init script. Do you want to run it?
         Yes

 ██████  Dependencies installing with npm...

      ▲  Oh no! Failed to install dependencies.
```

if I cd into the app-dir and run npm install I get the following output

```shell
› npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: appname@undefined
npm ERR! Found: @remix-run/react@2.4.0
npm ERR! node_modules/@remix-run/react
npm ERR!   @remix-run/react@"^2.4.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @remix-run/react@"^1.0.0" from remix-i18next@4.2.0
npm ERR! node_modules/remix-i18next
npm ERR!   remix-i18next@"^4.1.1" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR!
npm ERR! For a full report see:
npm ERR! C:\Users\user.name\AppData\Local\npm-cache\_logs\2023-12-21T10_52_51_177Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in: C:\Users\user.name\AppData\Local\npm-cache\_logs\2023-12-21T10_52_51_177Z-debug-0.log
```

--- a/remix.init/index.js
+++ b/remix.init/index.js
@@ -15,12 +15,8 @@ function getRandomString(length) {
 	return crypto.randomBytes(length).toString("hex");
 }
 
-async function main({ rootDirectory, packageManager, isTypeScript }) {
+async function main({ rootDirectory, packageManager }) {
 	console.log(`🚀  Making something cool with this template ...`);
-
-	if (!isTypeScript) {
-		throw new Error("😌  Sorry, this template only supports TypeScript");
-	}
 
 	const README_PATH = path.join(rootDirectory, "README.md");
 	const FLY_TOML_PATH = path.join(rootDirectory, "fly.toml");
@@ -78,7 +74,7 @@ async function main({ rootDirectory, packageManager, isTypeScript }) {
 		? dockerfile.replace(
 				new RegExp(escapeRegExp("ADD package.json"), "g"),
 				`ADD package.json ${lockfile}`,
-		  )
+			)
 		: dockerfile;
 
 	await Promise.all([

